File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] res=new int[2];
        Map<Integer,Integer> d=new HashMap<>();
        for(int i=0;i<nums.length;i++){
            int b=target-nums[i];
            if(d.containsKey(b)){
                res[0]=d.get(b);
                res[1]=i;
                return res;
            }
            else{
                d.put(nums[i],i);
            }
        }
        return res;
        
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], t: int) -> List[int]:
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j]==t:
                    return [i,j]

       

        